README.md

```diff
@@ -26,6 +26,7 @@ Deploy and log in with username *admin* and the password that's set up during de
  * Apache **2.4** or higher
  * MySQL >= 5.7 (or MariaDB >= 10.3) 
  * PHP <b>7.2</b> or higher
+ * NodeJS 6.17 or higher
```

Comment:
The README on the `minor` branch has version NodeJS 8.0. These versions should match.

Reply:
Hi John - this is for the next 21 `bugfix` release. 
Are you suggesting a new PR for minor, or does NodeJS 8.0 apply to Loris 21.0.* as well?

Reply:
8.0 applies everywhere.

Reply:
Ok thanks for catching this.  
This gives us an additional reason to do a bugfix release of 21.0.2